src/components/Emoji/Picker.js

```diff
@@ -0,0 +1,115 @@
+
+import { useEffect, useState, useContext } from "react";
+import { SmileIcon } from "../Icons";
+import { Picker } from "emoji-mart";
+import { ThemeContext } from "styled-components";
+import styled from "styled-components";
+import "emoji-mart/css/emoji-mart.css";
+
+const Wrapper = styled.span`
```

Comment:
Can you use this styling as an export and add it to `alpha/src/components/Tweet/EmojiTweet.js` as well - it will fix the picker box :)